docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md

```diff
@@ -0,0 +1,77 @@
+# Create Meta Block
+
+With the meta field registered in the previous step, next you will create a new block used to display the field to the user. See the [Block Tutorial](/docs/designers-developers/developers/tutorials/block-tutorial/readme.md) for a deeper understanding of creating custom blocks.
+
+For this block, you will use the TextControl component, this is similar to an HTML input text field. For additional components, see the list in the [editor package repository](https://github.com/WordPress/gutenberg/tree/master/packages/editor/src/components).
+
+Attributes are the information displayed in blocks. As shown in the block tutorial, the source of attributes come from the text/html a user writes in the editor. For your meta block, the attribute will come from the post meta field.
+
+By specifying the source of the attributes as `meta`, the Block Editor automatically handles the loading and storing of the data; no REST API or data functions are needed.
+
+Replace the previous content in `myguten.js` with the following:
```

Comment:
So far, the user hasn't been exposed to the `myguten.js` file. If the JavaScript tutorial is done just before this one, this will be just fine. There is a strong chance people don't read tutorials linearly, though, so I'd rather make the examples work in isolation as if the user only read this one.

Another thing that I'd fix is the plugin name header in the PHP file. It may be something obvious for seasoned WordPress developers. Newbies probably don't know that the header name is something necessary for the plugin to be listed in the plugins section.

Reply:
Good catch, an artifact from duplicating part of the JS tutorial and then changing to just link over. Updated both cases in https://github.com/WordPress/gutenberg/commit/179ee2950c98830ed0a33490f3989331eb17d905